src/basic/format-util.h

```diff
@@ -68,3 +69,9 @@
 #endif
 
 char *format_ifname(int ifindex, char buf[static IF_NAMESIZE + 1]);
+
+#define FORMAT_BYTES_MAX 8
+char *format_bytes_full(char *buf, size_t l, uint64_t t, bool use_iec, bool include_below_point, bool trailing_B);
```

Comment:
doesn't matter much, but the three bools would probably be nicer as single flags parameter